pinboard_config.example.rb

```diff
@@ -0,0 +1,63 @@
+{
+  secret_base_url: 'secret:abc123/',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.